ArduPlane/Log.cpp

```diff
@@ -286,10 +286,40 @@ const struct LogStructure Plane::log_structure[] = {
 // @Field: TAspd: target airspeed
     { LOG_NTUN_MSG, sizeof(log_Nav_Tuning),         
       "NTUN", "QfcccfffLLii",  "TimeUS,Dist,TBrg,NavBrg,AltErr,XT,XTi,AspdE,TLat,TLng,TAlt,TAspd", "smddmmmnDUmn", "F0BBB0B0GGBB" },
+
+// @LoggerMessage: SONR
+// @Description: Sonar (downward facing rangefinder) related messages. Used for altitude correction in Plane.
+// @Field: TimeUS: Microseconds since system startup
+// @Field: Dist: Distance as detected from the sensor
+// @Field: Volt: Voltage at the sensor pin (only applicable for analog sensor)
+// @Field: Cnt: Number of consecutive valid readings (maxes out at 10)
+// @Field: Corr: Correction made to altitude reading from barometer reading
     { LOG_SONAR_MSG, sizeof(log_Sonar),             
       "SONR", "QffBf",   "TimeUS,Dist,Volt,Cnt,Corr", "smv--", "FB0--" },
+
+// @LoggerMessage: ATRP
+// @Description: Pitch/Roll AutoTune messages 
```

Comment:
Some clarification that this is Plane's autotune not Copter-style autotune would be good.